### PR TITLE
`PropsWithClassName` interface is supported

### PR DESCRIPTION
**Pull request by sgrishchenko**

### CODE REVIEW

File: buildSrc/src/main/kotlin/com/github/turansky/react/MemberConverter.kt [hunk 1]
```diff
@@ -15,6 +15,8 @@ internal fun convertMembers(
 
     return content.removeSuffix(";")
         .splitToSequence(";\n")
+        // exclude className because it is inherited
+        .filter { !it.contains("className") }
```
Comment (turansky) [inline buildSrc/src/main/kotlin/com/github/turansky/react/MemberConverter.kt:19]: `startWith("className:")`?

**Review by turansky — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/kotlin/com/github/turansky/react/MemberConverter.kt (hunk 1 above)
Comment (sgrishchenko) [reply buildSrc/src/main/kotlin/com/github/turansky/react/MemberConverter.kt:19]: it does not work. It seems `SVGAttributes` starts with comment, so `className` is not filtered out for `SVGAttributes` and `SVGAttributes` breaks compilation.

Comment (turansky) [reply buildSrc/src/main/kotlin/com/github/turansky/react/MemberConverter.kt:19]: `"classname:" !in it`?

Comment (sgrishchenko) [reply buildSrc/src/main/kotlin/com/github/turansky/react/MemberConverter.kt]: done

**Pull request merged**